src/promptflow/tests/test_configs/flows/chat_group/chat_group_copilot/prompt.jinja2

```diff
@@ -0,0 +1,15 @@
+system:
```

Comment:
Seems there config files are referenced from existing flows that are in `test_configs/flows/`, why create a copy instead of using the original ones?

Reply:
I removed unnecessary properties, not exactly the same one. And I may have some update of it for different test